helm/cluster-openstack/values.yaml

```diff
@@ -72,3 +72,8 @@ nodePools: {}  # Node pools that should exist for the cluster. Each node pool mu
 includeClusterResourceSet: "true"  # must be a string
 # let's use the kubectl image defaults
 kubectlImage: {}
+
+# machine and certificate rotation prior to expiry
+rollout:
+  before: false
+  certExpiryDays: 0
```

Comment:
Just curious, Is there any reason why we set it to 0?

Reply:
Not really, I guess to write one less line in ice-clusters might as well set to our value, because it will only be used if `before` is true

Reply:
I see you have changed to 14 days. All is good to me :D